cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/stores/Store.wc.tsx

```diff
@@ -3,7 +3,7 @@ import React, { createContext, ReactNode, useReducer } from 'react';
 
 // App imports
 import { StoreDispatch } from '../../shared/types/types';
-import wcReducer from '../reducers/Wc.reducer';
+import wcReducer from '../reducers/Wizard.reducer';
```

Comment:
sorry this file confuses me a little bit. Is this just a quick cleanup change for the workload cluster store or were you intending to refactor and repurpose this store for something else?

Reply:
This is a quick cleanup. I'm seeing a proliferation of reducers that are all the same (a combined reducer of data and ui), and I'd like to eliminate the duplicate classes with a generic one